presto-orc/src/main/java/com/facebook/presto/orc/OrcReaderOptions.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import io.airlift.units.DataSize;
+
+import static io.airlift.units.DataSize.Unit.MEGABYTE;
+import static java.util.Objects.requireNonNull;
+
+public class OrcReaderOptions
+{
+    private final DataSize maxMergeDistance;
+    private final DataSize tinyStripeThreshold;
+    private final DataSize maxBlockSize;
+    private final boolean zstdJniDecompressionEnabled;
+
+    public OrcReaderOptions(DataSize maxMergeDistance, DataSize tinyStripeThreshold, DataSize maxBlockSize, boolean zstdJniDecompressionEnabled)
+    {
+        this.maxMergeDistance = requireNonNull(maxMergeDistance, "maxMergeDistance is null");
+        this.maxBlockSize = requireNonNull(maxBlockSize, "maxBlockSize is null");
+        this.tinyStripeThreshold = requireNonNull(tinyStripeThreshold, "tinyStripeThreshold is null");
+        this.zstdJniDecompressionEnabled = zstdJniDecompressionEnabled;
+    }
+
+    public DataSize getMaxMergeDistance()
+    {
+        return maxMergeDistance;
+    }
+
+    public DataSize getMaxBlockSize()
+    {
+        return maxBlockSize;
+    }
+
+    public boolean isOrcZstdJniDecompressionEnabled()
+    {
+        return zstdJniDecompressionEnabled;
+    }
+
+    public DataSize getTinyStripeThreshold()
+    {
+        return tinyStripeThreshold;
+    }
+
+    public static OrcReaderOptions createDefaultTestConfig()
```

Comment:
move this function to `OrcTestingUtils` for raptor. Inline or create separate helper functions in tests from other modules.